spec/traits.dd

```diff
@@ -658,8 +658,10 @@ $(H2 $(GNAME getOverloads))
 
         $(P The first argument is an aggregate (e.g. struct/class/module).
         The second argument is a string that matches the name of
-        one of the functions in that aggregate.
+        one of the members in that aggregate.
+        The third argument is a bool, and is optional.
         The result is a tuple of all the overloads of that function.
+        If the third member evaluates to true, the result will include all members that matches the supplied name.
```

Comment:
I recommend swapping this line with the one above it so the "The result..." is the last in the paragraph, and the explanation of the 3rd argument is not fragmented.

Reply:
Also, "the result will include all members that **match** the supplied name"